spec/features/adventure_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe "Adventures" do
+  let!(:adventure) { FactoryBot.create(:adventure) }
```

Comment:
can make this `{ create(:adventure) }` thanks to the `config.include FactoryBot::Syntax::Methods` line